### PR TITLE
fix privileged policy and add example gk policies

### PR DESCRIPTION
**Pull request by sozercan**
Signed-off-by: Sertac Ozercan <sozercan@gmail.com>

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1,10 +1,12 @@
 github.com/bytecodealliance/wasmtime-go v0.40.0 h1:7cGLQEctJf09JWBl3Ai0eMl1PTrXVAjkAb27+KHfIq0=
 github.com/bytecodealliance/wasmtime-go v0.40.0/go.mod h1:q320gUxqyI8yB+ZqRuaJOEnGkAnHh6WtJjMaT2CW4wI=
-github.com/edsrzf/mmap-go v1.0.0/go.mod h1:YO35OhQPt3KJa3ryjFM5Bs14WD66h8eGKpfaBNrHW5M=
-github.com/go-interpreter/wagon v0.6.0/go.mod h1:5+b/MBYkclRZngKF5s6qrgWxSLgE9F5dFdO1hAueZLc=
+github.com/josharian/intern v1.0.0 h1:vlS4z54oSdjm0bgjRigI+G1HpF+tI+9rE5LLzOg8HmY=
 github.com/josharian/intern v1.0.0/go.mod h1:5DoeVV0s6jJacbCEi61lwdGj/aVlrQvzHFFd8Hwg//Y=
+github.com/kubewarden/k8s-objects v1.24.0-kw3 h1:cRi2A5iESlZHqokXY0QXW6LdQaDXyLzTjja4V3Y2luo=
 github.com/kubewarden/k8s-objects v1.24.0-kw3/go.mod h1:IuIHLG1JtxjC1JnY7SyEEA9MukCh/FACcwpzaBjgdLQ=
+github.com/kubewarden/strfmt v0.1.2 h1:S0YUVkPeyUMikz8QssbMzfd1MC5K8ZqxLI2zfF8euMs=
```
Comment (ritazh) [inline go.sum]: do we need this?

Comment (sozercan) [reply go.sum]: i just run go mod tidy

Comment (sozercan) [reply go.sum]: i can move to gjson if you want but it'll work the same

Comment (ritazh) [reply go.sum]: what uses it? wdym by "move to gjson"

Comment (sozercan) [reply go.sum]: https://github.com/ritazh/hello-wasm/blob/main/allowedrepos/policy.go#L9
https://github.com/ritazh/hello-wasm/blob/main/allowedrepos/policy.go#L27

re: move to gjson, looks like this: https://github.com/ritazh/hello-wasm/blob/main/allowedrepos/policy.go#L25

Comment (ritazh) [reply go.sum]: oh! yea lets move to gjson

**Review by ritazh — APPROVED** [review]

**Pull request merged**